spec/features/shelters/show_spec.rb

```diff
@@ -76,6 +76,9 @@
     expect(page).to have_content("Shelter Pet Count: 2")
     expect(page).to have_content("Average Shelter Review: 4")
     expect(page).to have_content("Number of Applications: 2")
+    visit "/shelters/#{@shelter_2.id}"
```

Comment:
This was to ensure that our helper method was not counting all applications and only those for a specific shelter.